From 1301dd2bd9a692330135f69032cb26eab99c6540 Mon Sep 17 00:00:00 2001
From: Johannes Renner <jrenner@suse.com>
Date: Mon, 28 Aug 2017 15:19:58 +0200
Subject: [PATCH] Revert "We don't have python-systemd, so notify can't
 work"

This change can be reverted as notifying systemd should work now after
the merge of this pull request:

- https://github.com/openSUSE/salt/pull/33

This reverts commit 7917ec3a7efa9abda5932a0241070e0b048aa90d.
---
 pkg/salt-master.service | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pkg/salt-master.service b/pkg/salt-master.service
index ecd3edd467..b09a1003fa 100644
--- a/pkg/salt-master.service
+++ b/pkg/salt-master.service
@@ -4,7 +4,8 @@ After=network.target
 
 [Service]
 LimitNOFILE=16384
-Type=simple
+Type=notify
+NotifyAccess=all
 ExecStart=/usr/bin/salt-master
 TasksMax=infinity
 
-- 
2.13.6


